### ISSUE TITLE
gap:config-file not working

### ISSUE THREAD

**Issue by pastet89** [phonegap/build#647]
No matter what I try I cannot get  the ```gap:config-file``` to work. The build compiles without errors, but there are no changes in the AndroidManifest.xml.

This is my code:
```
   <gap:config-file platform="android" parent="/application">
        <meta-data android:name="google_analytics_adid_collection_enabled" android:value="false" />
    </gap:config-file>  
```

I tried also:
1) Putting it into the ```<platform name="android">``` node and out of it.
2) Tried: ```parent="/application"```,  ```parent="/manifest/application"```

**Comment by andersborgabiro** [comment]
This is how it's done now: https://cordova.apache.org/docs/en/latest/plugin_ref/spec.html#edit-config

**Comment by pastet89** [comment]
@andersborgabiro Thanks, however, it didn't work for me either. I spent few hours trying all combinations I could think of, tried both ```eidt-config``` and ```config-file```, various Xpath values, inside and outside the Android platform node, nothing works for me.

**Comment by andersborgabiro** [comment]
I've only used edit-config for iOS, so I don't have a good example. It no doubt works there.
Do you use cli-8.0.0?

**Comment by pastet89** [comment]
No, I use cli-7.0.1 and I build online through build.phonegap.com.

**Comment by andersborgabiro** [comment]
Any reason for doing that? cli-7.0.1 is very old. I use PGB too.

**Comment by pastet89** [comment]
Actually, yes. I can't remember exactly what was it, but there was some
kind of bug which required me to explicitly use that version. Anyway, I
will try to repack and edit AndroidManifest.xml using apktool, thanks for
your time!
